src/color.ts

```diff
@@ -0,0 +1,370 @@
+import type { Faker } from '.';
+import type cssFunctions from './locales/en/color/css_functions';
+import type cssSpaces from './locales/en/color/css_spaces';
+
+type CSSFunction = typeof cssFunctions[number];
+type CSSSpace = typeof cssSpaces[number];
+
+/**
+ * Formats the hex format of a generated color string according
+ * to options specified by user.
+ *
+ * @param hexColor Hex color string to be formated.
+ * @param options options object.
+ * @param options.prefix Prefix of the generated hex color. Defaults to `0x`.
+ * @param options.case Letter case of the generated hex color. Defaults to `mixed`.
+ */
+function applyHexFormat(
+  hexColor: string,
+  options?: {
+    prefix?: string;
+    case?: 'upper' | 'lower' | 'mixed';
+  }
+): string {
+  if (options?.prefix) hexColor = hexColor.replace('0x', options.prefix);
+  switch (options?.case) {
+    case 'upper':
+      hexColor = hexColor.toUpperCase();
+      break;
+    case 'lower':
+      hexColor = hexColor.toLowerCase();
+      break;
+  }
+  return hexColor;
+}
+
+/**
+ * Converts an array of numbers into binary string format.
+ *
+ * @param values Array of values to be converted.
+ */
+function toBinary(values: number[]): string {
+  const binary: string[] = values.map((value: number) => {
+    const isFloat: boolean = Number(value) === value && value % 1 !== 0;
+    if (isFloat) {
+      const buffer: ArrayBuffer = new ArrayBuffer(4);
```

Comment:
Type can be inferred
```suggestion
      const buffer = new ArrayBuffer(4);
```

Reply:
Done!